support-services/src/main/scala/com/gu/support/promotions/PromotionCache.scala

```diff
@@ -5,18 +5,16 @@ import org.joda.time.{DateTime, Minutes}
 
 import scala.concurrent.stm.{Ref, atomic}
 
-case class PromotionCache(fetched: DateTime, promotions: Iterable[Promotion]) {
-  def isFresh: Boolean = DateTime.now.getMillis - fetched.getMillis < PromotionCache.maxAge
+case class PromotionCacheResponse(fetched: DateTime, promotions: Iterable[Promotion]) {
+  def isFresh: Boolean = DateTime.now.getMillis - fetched.getMillis < Minutes.ONE.toStandardDuration.getMillis
```

Comment:
I like the extra documentation provided by the constant name, also this line is a bit verbose already
```suggestion
  val maxAge = Minutes.ONE.toStandardDuration.getMillis
  
  def isFresh: Boolean = DateTime.now.getMillis - fetched.getMillis < maxAge
```